app/models/kubernetes_image_scan.rb

```diff
@@ -0,0 +1,139 @@
+#
+# Copyright 2015 Red Hat, Inc.
+#
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
```

Comment:
Why is this whole license header here?  Why Lesser GPL, when our project is Apache licensed?